bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/gems/CandidateAsterismsTreeTableModel.java

```diff
@@ -95,9 +96,11 @@ public Object getValue(Row row) {
         // (XXX There might be another way to get the parent node)
         private Row _parent;
 
+        private final Option<Long> _when;
+
         // A row corresponding to a GemsGuideStars object
         // (top level node, displays the Strehl info and checkbox)
-        Row(GemsGuideStars gemsGuideStars, List<Row> children) {
+        Row(GemsGuideStars gemsGuideStars, List<Row> children, Option<Long> when) {
```

Comment:
Right. AGS has to pass a time to targets (otherwise it won't compile). But maybe the catalogs can do a better job if we pass the time to them as well. Don't know.